meta/1-3-1.md

```diff
@@ -11,7 +11,7 @@ un_designated_tier: 2.0
 un_custodian_agency: ILO
 goal_meta_link: https://unstats.un.org/sdgs/metadata/files/Metadata-01-03-01a.pdf
 goal_meta_link_text: United Nations Sustainable Development Goals Metadata (pdf 894kB)
-reporting_status: complete
+reporting_status: Выполнено
```

Comment:
Line 14 should be changed back to `reporting_status: complete` for this to show on the site